specifications/android.v2.md

```diff
@@ -0,0 +1,135 @@
+# Specifications
+
+UnifiedPush Spec: AND_2.0.0-beta2
+
+
+## Index
+
+* [Versions](#versions)
+* [Distributor Application](#distributor-application)
+* [Connector Library](#connector-library)
+* [Registration Broadcast Receiver](#registration-broadcast-receiver-1)
+* [Messaging Broadcast Receiver](#messaging-broadcast-receiver-1)
+
+
+## Versions
+
+Versions are identified on the distributor and the connector with the following 2 exposed actions:
```

Comment:
Maybe we should use permissions for this instead of actions.

https://developer.android.com/guide/topics/permissions/defining